packages/components/src/snackbar/list.tsx

```diff
@@ -40,27 +43,33 @@ const SNACKBAR_VARIANTS = {
 };
 
 const SNACKBAR_REDUCE_MOTION_VARIANTS = {
-	init: false,
-	open: false,
-	exit: false,
+	init: {},
+	open: {},
+	exit: {},
```

Comment:
Have you tried testing that these new values work as expected for `reducedMotion` users (i.e. the notices don't animate in/out) ?

A cleaner alternative to passing `{}` could be to remove the `SNACKBAR_REDUCE_MOTION_VARIANTS` variable altogether and simply pass `undefined` to the `variant` prop of `motion.div` when `isReducedMotion === true`

Reply:
>Have you tried testing that these new values work as expected for reducedMotion users (i.e. the notices don't animate in/out) ?

I just tested it, and it looks like they don't animate in/out:
![snackbarlist-no-animate](https://user-images.githubusercontent.com/4063887/199592516-f79dde99-8b0f-475b-9be4-6bd703738200.gif)

If it's alright, I just [added](https://github.com/WordPress/gutenberg/pull/45472/commits/28e6af69f13aff84b08c25f830fc6fc3c6073ef1) a story for `SnackbarList`.

Reply:
> A cleaner alternative to passing {} could be to remove the SNACKBAR_REDUCE_MOTION_VARIANTS variable altogether and simply pass undefined to the variant prop of motion.div when isReducedMotion === true

Thanks, that's much cleaner than keeping an object with 3 properties of `{}`.

Removed `SNACKBAR_REDUCE_MOTION_VARIANTS` in https://github.com/WordPress/gutenberg/pull/45472/commits/8667eb15b12f57617a935b49bda42ad44917f5fb

Reply:
> If it's alright, I just [added](https://github.com/kienstra/gutenberg/blob/9818ca5ff6b337fffb8eb2283916a572be12fe69/packages/components/src/snackbar/stories/list.tsx) a story for `SnackbarList`

Thank you! 

<details>

<summary>I went ahead and made a few additions locally, in order to be able to test the manual dismissal of the notices and their animation</summary>

```diff
diff --git a/packages/components/src/snackbar/stories/list.tsx b/packages/components/src/snackbar/stories/list.tsx
index d6567f3feb..a12b2c171a 100644
--- a/packages/components/src/snackbar/stories/list.tsx
+++ b/packages/components/src/snackbar/stories/list.tsx
@@ -3,6 +3,11 @@
  */
 import type { ComponentMeta, ComponentStory } from '@storybook/react';
 
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
@@ -29,9 +34,26 @@ export default meta;
 
 export const Default: ComponentStory< typeof SnackbarList > = ( {
 	children,
+	notices: noticesProp,
 	...props
 } ) => {
-	return <SnackbarList { ...props }>{ children }</SnackbarList>;
+	const [ notices, setNotices ] = useState( noticesProp );
+
+	const onRemove = ( id: string ) => {
+		const matchIndex = notices.findIndex( ( n ) => n.id === id );
+		if ( matchIndex > -1 ) {
+			setNotices( [
+				...notices.slice( 0, matchIndex ),
+				...notices.slice( matchIndex + 1 ),
+			] );
+		}
+	};
+
+	return (
+		<SnackbarList { ...props } notices={ notices } onRemove={ onRemove }>
+			{ children }
+		</SnackbarList>
+	);
 };
 
 Default.args = {
@@ -39,7 +61,7 @@ Default.args = {
 		'Use SnackbarList to communicate multiple low priority, non-interruptive messages to the user.',
 	notices: [
 		{
-			id: 'SAVE_POST_NOTICE_ID',
+			id: 'SAVE_POST_NOTICE_ID_1',
 			spokenMessage: 'Post published.',
 			actions: [
 				{
@@ -52,7 +74,7 @@ Default.args = {
 			explicitDismiss: false,
 		},
 		{
-			id: 'SAVE_POST_NOTICE_ID',
+			id: 'SAVE_POST_NOTICE_ID_2',
 			spokenMessage: 'Post updated',
 			actions: [
 				{
```
</details>

Let's review the Storybook examples later (maybe even in a different PRs) — but this is already a small improvement!

Reply:
Oh thanks! Committed this verbatim in https://github.com/WordPress/gutenberg/pull/45472/commits/fc26baf7fd0f852122728c92db0b17c98c10f0c8